src/cursor/cur_std.c

```diff
@@ -582,6 +595,100 @@ err:	API_END(session, ret);
 }
 
 /*
+ * __cursor_modify --
+ *	WT_CURSOR->modify default implementation.
+ */
+static int
+__cursor_modify(WT_CURSOR *cursor, WT_MODIFY *entries, int nentries)
```

Comment:
Just a general distaste, though I've been through this before. We use ``int`` a lot of places in our API, so please ignore me.